libr/search/search.c

```diff
@@ -427,7 +406,9 @@ R_API RList *r_search_find(RSearch *s, ut64 addr, const ut8 *buf, int len) {
 
 /* --- keywords --- */
 R_API int r_search_kw_add(RSearch *s, RSearchKeyword *kw) {
-	if (!kw) return false;
```

Comment:
an if doesnst hurts and prevents anoying null derefs, compilers can always optimize that stuff with lto